<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>738</width>
    <height>410</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>OS_CW</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,7,5">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QTableView" name="tableView_data">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>120</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="autoScrollMargin">
         <number>15</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QGraphicsView" name="graphicsView"/>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QSlider" name="horizontalSlider">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinBox">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>8</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QGraphicsView" name="graphicsView_2"/>
      </item>
      <item>
       <widget class="QTextEdit" name="text_errors"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>738</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="action_openFile"/>
    <addaction name="action_saveReport"/>
    <addaction name="action_openSettings"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Выполнить</string>
    </property>
    <addaction name="action_step_back"/>
    <addaction name="action_step_forward"/>
    <addaction name="action_play"/>
    <addaction name="separator"/>
    <addaction name="action_5"/>
    <addaction name="action_6"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>Справка</string>
    </property>
    <addaction name="action_openAbout"/>
    <addaction name="action_openHelp"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_openAbout">
   <property name="text">
    <string>О программе</string>
   </property>
  </action>
  <action name="action_openHelp">
   <property name="text">
    <string>&amp;Справка</string>
   </property>
  </action>
  <action name="action_openFile">
   <property name="text">
    <string>Открыть</string>
   </property>
  </action>
  <action name="action_step_back">
   <property name="text">
    <string>Шаг назад</string>
   </property>
  </action>
  <action name="action_step_forward">
   <property name="text">
    <string>Шаг вперед</string>
   </property>
  </action>
  <action name="action_play">
   <property name="text">
    <string>Авто-выполнение</string>
   </property>
  </action>
  <action name="action_5">
   <property name="text">
    <string>Повысить степень полинома</string>
   </property>
  </action>
  <action name="action_6">
   <property name="text">
    <string>Понизить степень полинома</string>
   </property>
  </action>
  <action name="action_4">
   <property name="text">
    <string>Отчёт</string>
   </property>
  </action>
  <action name="action_saveReport">
   <property name="text">
    <string>Сохранить отчёт</string>
   </property>
  </action>
  <action name="action_openSettings">
   <property name="text">
    <string>Настройки</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
